flink-dist/src/main/flink-bin/bin/taskmanager.sh

```diff
@@ -49,20 +49,33 @@ if [[ $STARTSTOP == "start" ]] || [[ $STARTSTOP == "start-foreground" ]]; then
 
     # Startup parameters
 
-    jvm_params=$(getTmResourceJvmParams)
+    jvm_params_output=`runBashJavaUtilsCmd GET_TM_RESOURCE_JVM_PARAMS ${FLINK_CONF_DIR}`
+    jvm_params=`extractExecutionParams "$jvm_params_output"`
     if [[ $? -ne 0 ]]; then
         echo "[ERROR] Could not get JVM parameters properly."
         exit 1
     fi
     export JVM_ARGS="${JVM_ARGS} ${jvm_params}"
 
     IFS=$" "
-    dynamic_configs=($(getTmResourceDynamicConfigs))
+
+    dynamic_configs_output=`runBashJavaUtilsCmd GET_TM_RESOURCE_DYNAMIC_CONFIGS ${FLINK_CONF_DIR}`
+    dynamic_configs=`extractExecutionParams "$dynamic_configs_output"`
```

Comment:
This is out of scope for this PR but I'm wondering why we need to call the utility twice. This will generate almost the same diagnostic output twice which will appear in the main process logs. I'd suggest to rework this as a follow up.

Reply:
I think this is a good suggestion.

Reply:
Tracked by: https://issues.apache.org/jira/browse/FLINK-15727